client/src/pages/locations/Show.tsx

```diff
@@ -60,9 +60,16 @@ const LocationShow = ({ pageDispatchers }: LocationShowProps) => {
   const routerLocation = useLocation()
   const stateSuccess = routerLocation.state?.success
   const [stateError, setStateError] = useState(routerLocation.state?.error)
+  const [attachments, setAttachments] = useState([])
   const { loading, error, data, refetch } = API.useApiQuery(GQL_GET_LOCATION, {
     uuid
   })
+  useEffect(() => {
+    if (data?.location) {
+      const location = new Location(data ? data.location : {})
+      setAttachments(location.attachments || [])
+    }
+  }, [data])
```

Comment:
Why not simply:
```javascript
  useEffect(() => {
    setAttachments(data?.location?.attachments || [])
  }, [data])
```
I would also suggest to put the `useEffect` after `usePageTitle`.